pc/src/main.rs

```diff
@@ -110,6 +93,45 @@ impl GameState {
 
 async fn run_game() {
 	let mut game_state = GameState::new();
+	let mut events = game_state.input_players.get_events();
+	let mut no_player = true;
+
+	match &mut game_state.game_run {
+		None => todo!("no game run"),
+		Some(ref mut game_run) => {
+			while no_player {
+				events = game_state.input_players.get_events();
+				for event in events {
+					if event.is_some() {
+						no_player = false;
+						break;
+					}
+				}
+				next_frame().await;
+			}
+
+				//let players = game_state.input_players;
+				for player in &game_state.input_players.possible_players {
```

Comment:
you does iterator over `.possible_players`, this are player which are which does not join the game yet or devices which are no pybadge. Iterate over `.players` instead.